subsys/bluetooth/mesh/Kconfig

```diff
@@ -170,6 +170,19 @@ config BT_MESH_RELAY_ADV_SETS
 	  Maximum of simultaneous relay message support. Requires controller support
 	  multiple advertising sets.
 
+config BT_MESH_BRG_ADV_SETS
```

Comment:
It is quite cumbersome. I set this parameter to correct value and forget about this. Everything works.
After a couple of months, I enable relay feature, and my subnetwork bridge starts dropping messages due to usage of another set of Kconfig options.

Reply:
`BT_MESH_RELAY_BUF_COUNT` is under "if `BT_MESH_RELAY`, and `BT_MESH_RELAY_ADV_SETS` depends on it, so as I mentioned to @omkar3141 above we would need to extract and place them somewhere commonly accessible for both the relay and brg module